### DISCUSSION TITLE
[FR] Toggle number of grid

### DISCUSSION THREAD

**Discussion by stevenlis** [rxhanson/Rectangle#887] [Ideas]
Hi @rxhanson. First of all, I really would like to thank you for this amazing app.

I'm having some issues with "Snap windows by dragging" and wonder if you could pls consider adding a toggle for different grid pattern. 

I'm connecting my laptop to a vertical monitor and when I drag a window to snap, it sometimes activates the first third... sometimes the first two thirds...sometimes the center third... sometimes top half... Since the app supports so many different grid patterns, sometimes it's very hard to find the sweet spot to activate the desired grid.

Thus, I wonder if you could add a toggle. For example, a (2 by 1 or 1 by 2) grid will only allow me to snap windows to the top half or bottom half when I drag windows to snap.

**Comment by rxhanson** [comment]
Thanks for the feedback. I'm currently working on allowing users to configure the drag to snap areas. After the next release is out with that functionality, let me know if it's what you're looking for or not.

**Reply by stevenlis** [reply]
Thanks man. But I'm not sure at this moment, but I will get it a shot once it comes out.
Btw, I was setting up Raycast today and another idea might be just adding toggles for each option instead of grid pattern
 
![CleanShot 2022-08-21 at 11 03 46](https://user-images.githubusercontent.com/33796896/185797505-4eec1519-4f51-4c25-8806-d25b39aac19d.png)
